### ISSUE TITLE
Markdown Importer

### ISSUE THREAD

**Issue by adamziel** [adamziel/playground-docs-workflow#51]
[WordPress for Docs](https://github.com/WordPress/wordpress-playground/issues/1544) require a markdown importing pipeline. Let's integrate one with Playground, see also https://github.com/adamziel/site-transfer-protocol

Furthermore, migrating existing projects into Playground/based workflow would be much easier if there was a markdown importer — upload a zipped directory structure with markdown files inside (optionally including frontmatter), and get a hierarchy of WordPress pages with valid blocks.

Cc @jordesign as This sounds like a data liberation project.

Let's also do this while we are working on git imports https://github.com/WordPress/playground-tools/issues/314

**Comment by jordesign** [comment]
This kind of thing would be brilliant!

What kind of structure would/could something like this look like within the ZIP file? 
* directories for pages/sub-pages?
* Inclusion of linked media somehow?

**Comment by adamziel** [comment]
Yes and yes :) also markdown hrefs would be preserved as WordPress links

**Comment by jordesign** [comment]
I like this _a lot_ !

**Comment by carstingaxion** [comment]
> directories for pages/sub-pages?

I'd like to add another level, the root level should be post_type followed by pages/subpages.

That would allow to import any kind of content, may it be deeply nested hierarchical pages or posts with no hierarchy at all. Or whatever custom post type someone's (documentation) website would need.
